src/components/ContentWidgets/ToDoList/ToDoList.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { ContentSingularData } from '../../_data/ContentSingularData';
+import './ToDoList.css';
+
+export type ToDoListItem = {
+    toDoListItem_content: string
+    toDoListItem_completed: boolean
+    toDoListItem_order: number
+}
+
+export const ToDoList: React.FC<ContentSingularData> = ({
+    toDoList_content
+}) => {
+    if (!toDoList_content) {
+        return <></>
+    }
+
+    let toDoListItems = toDoList_content.map(item => {
```

Comment:
Rather than letting a variable be set to this, I would just put it within the `ul` tag you return.